src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/VilkårServiceTest.kt

```diff
@@ -72,50 +82,72 @@ internal class VilkårServiceTest {
         every { vilkårGrunnlagService.hentGrunnlag(behandlingId) } returns mockVilkårGrunnlagDto()
     }
 
-    /*
-    @Test
-    fun `skal opprette nye Vilkår for barnetilsyn med alle vilkår dersom ingen vurderinger finnes`() {
-        every { vilkårsvurderingRepository.findByBehandlingId(behandlingId) } returns emptyList()
-        every { fagsakService.hentFagsakForBehandling(behandlingId) } returns fagsak()
+    @Nested
+    inner class OppretteVilkårBarnetilsyn {
+        @BeforeEach
+        fun setUp() {
+            every { vilkårRepository.findByBehandlingId(behandlingId) } returns emptyList()
+            every { fagsakService.hentFagsakForBehandling(behandlingId) } returns fagsak()
+        }
 
-        val nyeVilkårsvurderinger = slot<List<Vilkårsvurdering>>()
-        every { vilkårsvurderingRepository.insertAll(capture(nyeVilkårsvurderinger)) } answers { firstArg() }
-        val vilkår = VilkårType.hentVilkårForStønad(Stønadstype.BARNETILSYN)
+        @Test
+        fun `skal opprette nye Vilkår for barnetilsyn med alle vilkår dersom ingen vurderinger finnes`() {
+            val nyttVilkårsett = slot<List<Vilkår>>()
+            every { vilkårRepository.insertAll(capture(nyttVilkårsett)) } answers { firstArg() }
```

Comment:
Disse går å trekke ut fra alle tester, noe i stil med:
(tror det skal gå fint, hvis ikke finnes slot på ande steder)
```kotlin

        private val nyttVilkårsett = slot<List<Vilkår>>()
        @BeforeEach
        fun setUp() {
            every { vilkårRepository.insertAll(capture(nyttVilkårsett)) } answers { firstArg() }

```

Reply:
👏 for fine tester